search-sentiments-map/public/scripts/map-script.js

```diff
@@ -107,17 +111,28 @@ function loadCountryData() {
  * @returns {googe.maps.Data.StyleOptions} styling information for feature
  */
 function styleFeature(feature) {
-  let low = [5, 69, 54];  // Color of smallest datum.
-  let high = [151, 83, 34]; // Color of largest datum.
-
-  // Delta represents where the value sits between the min and max.
-  let delta = (feature.getProperty('country_data') - dataMin) /
-      (dataMax - dataMin);
-
+  let high = CountryColorCodes.RED;
+  let low = CountryColorCodes.GREEN;
   let color = [];
-  for (let i = 0; i < 3; i++) {
-    // Calculate an integer color based on the delta.
-    color[i] = (high[i] - low[i]) * delta + low[i];
+  let countryData = feature.getProperty('country_data');
+
+  if (countryData == null) {
+    // Set country color to be light gray if that country is disabled(occurs in
+    // user search).
+    color = CountryColorCodes.LIGHT_GRAY;
+  } else if (countryData === NO_RESULTS_DEFAULT_SCORE) {
+    // Set country color to be dark gray if that country has no results.
+    color = CountryColorCodes.DARK_GRAY;  
+  } else if (countryData != null) {
+    // Delta represents where the value sits between the min and max.
+    let delta = (countryData - DATA_MIN) /
+        (DATA_MAX - DATA_MIN);
+
+    color = [];
+    for (let i = 0; i < 3; i++) {
+      // Calculate an integer color based on the delta.
```

Comment:
Move and edit this comment to above for loop ... i.e. Calculate rgb integer values based on detla